browse/services/database/__init__.py

```diff
@@ -170,94 +171,97 @@ def get_recent_trackback_pings(max_trackbacks: int = 25) \
         .subquery()
     )
     tb_doc_tup = (
-        db.session.query(TrackbackPing, Document.paper_id, Document.title)
+        select(TrackbackPing, Document.paper_id, Document.title)
         .join(Document, TrackbackPing.document_id == Document.document_id)
         .filter(TrackbackPing.status == "accepted")
         .filter(TrackbackPing.url == stmt.c.url)
         .order_by(TrackbackPing.posted_date.desc())
-        .all()
     )
 
-    return list(tb_doc_tup)
+    return [row._t for row in session.execute(tb_doc_tup).all()]
 
 
 #Used on abs page
 @db_handle_error(db_logger=logger, default_return_val=None)
 def get_trackback_ping_latest_date(paper_id: str) -> Optional[datetime]:
     """Get the most recent accepted trackback datetime for a paper_id."""
-    timestamp: int = db.session.query(func.max(TrackbackPing.approved_time)).filter(
-        TrackbackPing.document_id == Document.document_id
-    ).filter(Document.paper_id == paper_id).filter(
-        TrackbackPing.status == "accepted"
+    timestamp = session.execute(
+        select(func.max(TrackbackPing.approved_time))
+        .filter(TrackbackPing.document_id == Document.document_id)
+        .filter(Document.paper_id == paper_id)
+        .filter(TrackbackPing.status == "accepted")
     ).scalar()
-    dt = datetime.fromtimestamp(timestamp, tz=tz)
-    dt = dt.astimezone(tz=tzutc())
-    return dt
+    if timestamp:
+        dt = datetime.fromtimestamp(timestamp, tz=tz)
+        dt = dt.astimezone(tz=tzutc())
+        return dt
+    return None
 
 
 # used on abs page
 @db_handle_error(db_logger=logger, default_return_val=0)
```

Comment:
I remember Brian C saying something about not handling errors and letting flask do it. I know these were already here, but I wonder if the refactor would be a good time to look into it